src/Packetery/Module/Order/shared/SharedOrderDetailsFormFactory.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Class SharedOrderDetailsForm.
+ *
+ * @package Packetery
+ */
+
+declare( strict_types=1 );
+
+namespace Packetery\Module\Order\Shared;
+
+use Packetery\Module\FormFactory;
+use Packetery\Module\FormValidators;
+use Packetery\Core\Helper;
+use Packetery\Nette\Forms\Form;
+
+/**
+ * Class SharedOrderDetailsForm.
+ *
+ * @package Packetery
+ */
+class SharedOrderDetailsFormFactory {
```

Comment:
Proč se tato třída nejmenuje jen FormFactory v namespace Packetery\Module\Order ?

Reply:
Chtěl jsem, aby z názvu jasně vyplývalo, na co se ta classa používá. Jelikož v zadání bylo vytvoř společný formulář pro modal a metabox, tak jsem si myslel, že to bude použito pouze na tyto dva případy.

Reply:
Změna provedena.